labelme2coco/cli.py

```diff
@@ -5,7 +5,7 @@
 
 def app() -> None:
     """Cli app."""
-    fire.Fire(convert)
+    fire.Fire(lambda labelme_folder, export_dir="runs/labelme2coco/", train_split_rate=1, skip_labels=[], category_id_start=0: convert(labelme_folder, export_dir, train_split_rate, skip_labels, category_id_start))
```

Comment:
This change is unnecessary. Once you revert this change , we can merge it 👍

Reply:
Changed it back